src/components/vm/overview/bootOrder.tsx

```diff
@@ -223,7 +263,7 @@ class BootOrderModal extends React.Component {
         this.moveDown = this.moveDown.bind(this);
     }
 
-    dialogErrorSet(text, detail) {
+    dialogErrorSet(text: string, detail: string) {
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-2185-af23a7d6-20250716-112918-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)